includes/Core/Assets/Assets.php

```diff
@@ -766,8 +766,12 @@ private function get_inline_user_data() {
 	 * @return array The dashboard sharing inline data to be output.
 	 */
 	private function get_inline_dashboard_sharing_data() {
-		$all_roles   = wp_roles()->roles;
-		$inline_data = array( 'roles' => array() );
+		$inline_data = array(
+			'roles' => array(),
+		);
+
+		// Expose roles to the client.
+		$all_roles = wp_roles()->roles;
```

Comment:
I think these changes aren't needed anymore. Let's revert it.

Reply:
@eugene-manuilov Do you mean we don't need to return `roles` array at all to the client?